.travis.yml

```diff
@@ -50,13 +53,17 @@ install:
   # Install the slycot package (two ways, to improve robustness).  For the 
   # conda version, need to install lapack from conda-forge (no way to specify 
   # this in the recipe).
+  # add the conda-forge channel to the config, otherwise openblas or
+  # lapack cannot be found in the check
+  # with --override-channels to make sure the locally built slycot is installed
   #
   - if [[ $TEST_CONDA == 1 ]]; then
-      conda install -c conda-forge lapack;
+      conda config --append channels conda-forge;
       conda build --python "$TRAVIS_PYTHON_VERSION" conda-recipe;
-      conda install slycot --use-local;
+      conda install -c conda-forge lapack;
+      conda install --override-channels -c local slycot;
```

Comment:
Why this way of doing a local install instead of using ``--use-local`` flag?  I believe ``--use-local`` is what is in the standard conda recipe template.

Reply:
Yes. But --use-local *adds* the local channel to your set of channels. If the slycot build failed, it would happily install the slycot found in other channels! This happened in the python 2.7 and 3.5 variants.
Since in the overviews of installed modules I saw "local" listed as a channel for the locally built slycot, I assumed that that channel was simply called local. And it works. 

Also, --override-channels, which you need to block all others, will not work with --use-local.